components/phpunit_bridge.rst

```diff
@@ -759,11 +759,25 @@ classes, interfaces and/or traits for the code to run::
 
     class MyClassTest extends TestCase
     {
+        public static function setUpBeforeClass(): void
+        {
+            ClassExistsMock::register(MyClass::class);
+        }
```

Comment:
If you are using `setUpBeforeClass` you also need to use `tearDownAfterClass`:
```php
    public static function tearDownAfterClass(): void
    {
        ClassExistsMock::withMockedClasses([]);
    }
```

But usign `setUp()`, `tearDown()` and `*BeforeClass`, `*AfterClass` seems not like a real world use case 🤔